src/sora-card.md

```diff
@@ -0,0 +1,71 @@
+
+### How do I know if I’m using a self-custodial wallet?
+
+All self-custodial crypto wallets enable you—and only you—to possess the private key associated with your public address. This private key takes the form of either a file or a “mnemonic phrase” consisting of 12-24 randomly generated words. If your wallet does not have this option, it is a custodial wallet and you are not in full control of your crypto assets!
+
+## What is a DEX?
+
+Decentralized exchanges, also known as DEXs, are peer-to-peer marketplaces where cryptocurrency traders make transactions directly on-chain without handing over management of their funds to an intermediary or custodian. Polkaswap.io is a great example of a well-designed and efficient DEX. Executing transactions directly with a blockchain makes DEXs the ultimate in financial freedom and privacy. These transactions are facilitated through the use of self-executing agreements written in code called smart contracts.
+
+Because DEXs allow users to trade directly from their wallets by interacting with the blockchain, users must guard their funds and are responsible for losing them if they make mistakes such as losing their private keys or sending funds to the wrong addresses.
+
+On the other hand, centralized exchanges (CEXs) are managed by a centralized organization such as a bank or exchange (e.g. Binance), that are involved in financial services looking to make a profit, and they can also become insolvent and lose all customer funds. High regulation of centralized exchanges often makes them harder to onboard, as many questions and personal information must be acquired before users can trade.
+
+SORA Card strives to offer the ease-of-use and self-custodial empowerment that users of Polkaswap and the SORA network expect, with links to the global financial system that traditionally were the purview of centralized exchanges.
+
+## How does SORA Card ensure the highest privacy and security standards?
+
+On a DEX or self-custodial wallet, users can immediately access their crypto balances after connecting with their private key. They will not be required to submit any personal information like names and addresses. This is great for individuals who value their privacy and personal sovereignty over their own assets.
+
+When registering an IBAN account, KYC is required because this is linked to the TradFi world. To safeguard your privacy as much as possible even when setting up an IBAN, the integrated KYC happens directly at the IBAN provider and none of your personal data is ever stored on the SORA network, or the SORA Wallet. Furthermore, the IBAN provider does not have insight into the transactions of your crypto wallet. In this way, your crypto wallet remains self-custodial, anonymous, and decentralized, while your fiat wallet abides by regulations and gives you access to the TradFi global financial system.
```

Comment:
```suggestion
When registering an IBAN account, KYC is required because this is linked to the TradFi world. To safeguard your privacy as much as possible even when setting up an IBAN, the integrated KYC happens directly at the IBAN provider and none of your personal data is ever stored on the SORA network, or the SORA Wallet. Furthermore, the IBAN provider does not have insight into the transactions of your crypto wallet. This way, your crypto wallet remains self-custodial, anonymous, and decentralized, while your fiat wallet abides by regulations and gives you access to the TradFi global financial system.
```